subxt/src/backend/unstable/mod.rs

```diff
@@ -496,15 +498,40 @@ impl<T: Config + Send + Sync + 'static> Backend<T> for UnstableBackend<T> {
     async fn stream_finalized_block_headers(
         &self,
     ) -> Result<StreamOfResults<(T::Header, BlockRef<T::Hash>)>, Error> {
-        let stream = self
```

Comment:
would be nice to unify with stream_headers such that closure could used in FollowStreamFinalizedHeads but can't use such thing since we need the state to decide how to deal with it.